plugins/pom.xml

```diff
@@ -20,6 +20,7 @@
                 <module>org.locationtech.udig.catalog.geotiff.tests</module>
                 <module>org.locationtech.udig.catalog.jgrass.tests</module>
                 <module>org.locationtech.udig.catalog.tests</module>
+                <module>org.locationtech.udig.catalog.tests2</module>
```

Comment:
Why a second module?

Reply:
catalog.tests are testing that require to fire up OSGi, catalog.test2 is to run mocked tests without OSGi. Do you habe a better idea for the module name?